code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -0,0 +1,285 @@
+	to_chat(usr, "<span class='notice'>Force Push</span>: Move things around with the power of the Force.")
+	to_chat(usr, "<span class='notice'>Light Saber Training</span>: You can deflect most projectiles and melee attacks while holding an activated energy sword.")
+
+
+///Triggers on COMSIG_MOB_ATTACK_RANGED. Usually handles stuff like picking up items at range.
+/datum/martial_art/starter_sith/proc/on_ranged_attack(datum/source, atom/target)
+	if(owner.dna.check_mutation(TK)) //FULPSTATION Chaplain Starter Sith PR by Surrealistik Jan 2020; no stacking with Telekinesis
+		return
+	target.attack_tk(owner)
+
+/datum/martial_art/starter_sith/teach(mob/living/carbon/human/H, make_temporary = FALSE)
+	. = ..()
+	if(!.)
+		return
+	owner = H
+	RegisterSignal(H, COMSIG_MOB_ATTACK_RANGED, .proc/on_ranged_attack)
+
+
+/datum/martial_art/starter_sith/on_remove(mob/living/carbon/human/H)
+	. = ..()
+	UnregisterSignal(H, COMSIG_MOB_ATTACK_RANGED)
+
+/obj/item/melee/transforming/energy/sword/saber/sith
+	name = "Sith Lightsaber"
+	possible_colors = list("red" = LIGHT_COLOR_RED)
+	desc = "A deadly and ancient energy sword hailing from an equally deadly and ancient religion."
+	armour_penetration = 100 //Cuts through anything.
+	force_on = 35 //Slightly better than standard.
+	throwforce_on = 30 //Better than standard.
+
+/obj/item/melee/transforming/energy/sword/saber/sith/IsReflect(mob/living/user)
+	if(!user.mind.has_martialart(MARTIALART_STARTERSITH))
+		return FALSE
+
+	if (!user.incapacitated() && prob(FORCETRAINING_BLOCKCHANCE)) //30% chance to reflect if we have sith training
+		spark_system.start()
+		playsound(src, pick('sound/weapons/blade1.ogg'), 75, TRUE)
+		return TRUE
+
+/obj/item/melee/transforming/energy/sword/proc/deflect_check(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(owner.mind.has_martialart(MARTIALART_STARTERSITH))
+		final_block_chance += FORCETRAINING_BLOCKCHANCE //+30% for having Sith training
+	SEND_SIGNAL(src, COMSIG_ITEM_HIT_REACT, args)
+	if(prob(final_block_chance))
+		spark_system.start()
+		playsound(src, pick('sound/weapons/blade1.ogg'), 75, TRUE)
+		owner.visible_message("<span class='danger'>[owner] deflects [attack_text] with [src]!</span>")
+		return TRUE
+	return FALSE
+
+
+/obj/item/melee/transforming/energy/sword/proc/spark_setup()
+	spark_system = new /datum/effect_system/spark_spread()
+	spark_system.set_up(5, 0, src)
+	spark_system.attach(src)
+
+
+
+/obj/item/book/granter/proc/sith_one_use(mob/user)
+	if(!oneuse)
+		return FALSE
+	var/datum/effect_system/spark_spread/spark_system = new /datum/effect_system/spark_spread()
+	spark_system.set_up(5, 0, src)
+	spark_system.attach(src)
+	spark_system.start()
+	if(isliving(user))
+		var/mob/living/L = user
+		L.electrocute_act(5,"Force Lightning",1,SHOCK_NOSTUN) //Snappy zappy.

```

Comment:
After playing around with it I feel the cooldown for the spell is too quick and people can end up spamming it.

Reply:
See above.